### ISSUE TITLE
test_gdasapp_convert_bufr_adpsfc_snow does not run

### ISSUE THREAD

**Issue by CoryMartin-NOAA** [NOAA-EMC/GDASApp#573]
The `feature/stable-nightly` branch won't update because one of the tests, `test_gdasapp_convert_bufr_adpsfc_snow` does not run. It complains:
```
Could not find executable /scratch1/NCEPDEV/da/Cory.R.Martin/CI/GDASApp/stable/20230814/global-workflow/sorc/gdas.cd/test/land/test_bufr2ioda.sh 
```

Either this needs fixed, or the test removed.

**Comment by jiaruidong2017** [comment]
@CoryMartin-NOAA I made and committed the changes for the G-W, but forgot to update the changes in the GDASApp.

**Comment by CoryMartin-NOAA** [comment]
Closed by #574
